projects/igniteui-angular/src/lib/navigation-drawer/themes/shared/_indigo.scss

```diff
@@ -0,0 +1,42 @@
+@use 'igniteui-theming/sass/typography' as *;
+@use 'igniteui-theming/sass/bem' as *;
+@use 'styles/themes/standalone' as *;
+
+@include themed-block(igx-nav-drawer, indigo) {
+    --aside-padding: #{rem(8px)};
+    --item-padding: #{rem(8px) rem(16px)};
+    --item-gap: #{rem(16px)};
+    --item-min-height: #{rem(32px)};
+    --igx-nav-drawer-size--mini: #{rem(48px)};
+
+    @include e(item) {
+        @include type-style('subtitle-2');
+
+        margin-block-end: rem(4px);
+
+        igx-icon {
+            --component-size: 2;
+        }
+    }
+
+    @include e(item, active) {
+        igx-icon {
+            --component-size: 2;
+        }
+    }
+
+    @include e(item, header) {
+        @include type-style('overline');
+
+        margin-block-end: rem(4px);
+    }
+
+    @include e(aside, mini) {
+        .igx-nav-drawer__item,
+        .igx-nav-drawer__item--active {
+            width: rem(32px);
+
+            margin: rem(4px) auto;
+        }
```

Comment:
Same as my previous comment. Consider making the width and margin of the item CSS custom properties. For easier modification between themes and to reduce the amount of produced CSS.